cloud-consumption-api/src/main/java/com/sequenceiq/consumption/api/v1/consumption/model/request/StorageConsumptionRequest.java

```diff
@@ -16,19 +17,33 @@ public class StorageConsumptionRequest extends ConsumptionBaseRequest {
     @ApiModelProperty(value = ConsumptionModelDescription.STORAGE_LOCATION, required = true)
     private String storageLocation;
 
+    @ApiModelProperty(value = ConsumptionModelDescription.CONSUMPTION_TYPE, allowableValues = "UNKNOWN,STORAGE,EBS")
+    private ConsumptionType consumptionType;
+
+    public ConsumptionType getConsumptionType() {
+        return consumptionType;
+    }
+
+    public void setConsumptionType(ConsumptionType consumptionType) {
+        this.consumptionType = consumptionType;
+    }
+
     public String getStorageLocation() {
+
         return storageLocation;
```

Comment:
I think, if you extend with a new value and use it in the new API endpoint, it would be better choice because later it hard to change. cc: @doktoric

Reply:
I am not sure about this as it may cause API compatibility issue

Reply:
what @topolyai5 mentioned is just an additional field so we can add that.

Reply:
I have talked with Ricsi, and the best approach if you create a new request class based on this but replace the storage location with cloudResourceId